microsoft/testsuites/ltp/ltp.py

```diff
@@ -52,30 +54,58 @@ class Ltp(Tool):
     LTP_DIR_NAME = "ltp"
     DEFAULT_LTP_TESTS_GIT_TAG = "20230929"
     LTP_GIT_URL = "https://github.com/linux-test-project/ltp.git"
+    LTP_RESULT_FILE = "ltp-results.log"
+    LTP_OUTPUT_FILE = "ltp-output.log"
+    LTP_SKIP_FILE = "skipfile"
     BUILD_REQUIRED_DISK_SIZE_IN_GB = 2
-    LTP_RESULT_PATH = "/opt/ltp/ltp-results.log"
-    LTP_OUTPUT_PATH = "/opt/ltp/ltp-output.log"
-    LTP_SKIP_FILE = "/opt/ltp/skipfile"
     COMPILE_TIMEOUT = 1800
     RUN_TIMEOUT = 12000
 
     @property
     def command(self) -> str:
-        return "/opt/ltp/runltp"
+        return self._command_path + "runltp"
 
     @property
     def dependencies(self) -> List[Type[Tool]]:
-        return [Make, Gcc, Git]
+        if self._prebuild_package:
+            return []
+        else:
+            return [Make, Gcc, Git]
 
     @property
     def can_install(self) -> bool:
         return True
 
-    def __init__(self, node: Node, *args: Any, **kwargs: Any) -> None:
+    @property
+    def result_path(self) -> str:
+        return self._command_path + self.LTP_RESULT_FILE
+
+    @property
+    def output_path(self) -> str:
+        return self._command_path + self.LTP_OUTPUT_FILE
+
+    @property
+    def skip_path(self) -> str:
+        return self._command_path + self.LTP_SKIP_FILE
+
+    def _check_exists(self) -> bool:
+        return (
+            len(self.node.tools[Ls].list(str(self._command_path), sudo=True)) > 0
+        )
+
+    def __init__(self, node: Node, prebuild_package: str, *args: Any, **kwargs: Any) -> None:
         super().__init__(node, args, kwargs)
         git_tag = kwargs.get("git_tag", "")
         self._git_tag = git_tag if git_tag else self.DEFAULT_LTP_TESTS_GIT_TAG
 
+        self._prebuild_package = prebuild_package
+        if self._prebuild_package:
+            self._command_path = "/tmp/ltp/"
+            self._remote_tar_path = self.node.get_pure_path(self._command_path) / os.path.basename(self._prebuild_package)
```

Comment:
The two variables can be set or use when installing. They don't need to be saved in class.

Reply:
The change is done, thanks